examples/minimal/astro.config.mjs

```diff
@@ -9,5 +9,5 @@
 // @ts-check
 export default /** @type {import('astro').AstroUserConfig} */ ({
   // Set "renderers" to "[]" to disable all default, builtin component support.
-  renderers: [],
+  // renderers: [],
```

Comment:
Muted these

Reply:
this one is actually intentional! I just push a commit to rewrite to make this more clear

Reply:
ah, i see, sorry for that man